.solhint.json

```diff
@@ -1,22 +1,20 @@
 {
   "extends": "solhint:recommended",
   "rules": {
+    "avoid-low-level-calls": "off",
     "compiler-version": ["off"],
     "constructor-syntax": "warn",
-    "quotes": ["error", "single"],
     "func-visibility": ["warn", { "ignoreConstructors": true }],
-    "not-rely-on-time": "off",
-    "no-inline-assembly": "off",
+    "no-console": "off",
```

Comment:
should we turn this on?

Reply:
I think so. Shoud we however disable it for `script/` or `test/`?

Reply:
for test perhaps, tho is where it appears the most, tests shouldn't have console logs, perhaps scripts do, but indifferent for now

Reply:
enabled `no-console` but also other rules that seemed reasonable and were not currently causing any errors 
- `no-inline-assembly` (not really for this but I don't see this current implementation requiring inline assembly)
- `no-console`
- `compiler-version`
- `no-empty-blocks`